tests/test_version.py

```diff
@@ -8,7 +8,7 @@ def check_accepted_version(monero, valid_version: bytes):
     major, minor, patch = monero.reset_and_get_version(
         monero_client_version=valid_version
     )  # type: int, int, int
-    assert (major, minor, patch) == (1, 9, 0)  # version of the Monero app
+    assert (major, minor, patch) == (2, 0, 0)  # version of the Monero app
```

Comment:
Could be usefull to get it directly from the Makefile: https://github.com/LedgerHQ/app-ethereum/blob/76e8690a68fb84910d09fbfd14cc975cefabe41a/tests/ragger/test_configuration_cmd.py#L9

Reply:
ok 👍